Jenkinsfile

```diff
@@ -3040,7 +3040,7 @@ pipeline {
         model.hidden_size=64 \
         model.num_attention_heads=8 \
         model.activation='swiglu' \
-        model.bias_gelu_fusion=False \
+        model.bias_activation_fusion=False \
```

Comment:
I assume this isn't backward compatible change, try to reduce such breaking changes if possible.